store/sqlstore/group_store.go

```diff
@@ -132,6 +133,15 @@ func (s *SqlGroupStore) Get(groupId string) store.StoreChannel {
 	})
 }
 
+func (s *SqlGroupStore) GetByIDs(groupIDs []string) ([]*model.Group, *model.AppError) {
+	var groups []*model.Group
+	query := fmt.Sprintf("SELECT * from UserGroups WHERE Id IN ('%s')", strings.Join(groupIDs, "', '"))
```

Comment:
Can we use here the squirrel query builder. It solves pretty well the "IN" clauses with lists, and I'm not sure if this change can derive in a sql injection problem, you are concatenating arbitrary strings with ",", and building it into the query with interpolation. Sounds like dangerous to me.

Reply:
Good idea.